onlinejudge/_implementation/utils.py

```diff
@@ -50,6 +50,26 @@ def next_sibling_tag(tag: bs4.Tag) -> bs4.Tag:
     return tag
 
 
+# remove all HTML tag without interpretation (except <br>)
+# remove all comment
+# using DFS(Depth First Search)
```

Comment:
```suggestion
# using DFS(Depth First Search)
# discussed in https://github.com/kmyk/online-judge-tools/issues/553
```

I want to add a link to the issue, here too, because it's difficult to understand why `tag.string` doesn't work.